cobra/flux_analysis/summary.py

```diff
@@ -167,10 +164,9 @@ def model_summary(model, threshold=1E-8, fva=None, digits=2):
                               'err': half_span.loc[r.id]}
              for r in in_rxns}).T
 
-        out_fluxes.sort_values(by='x', ascending=False, inplace=True)
-        out_fluxes = out_fluxes.round(digits)
-        in_fluxes.sort_values(by='x', inplace=True)
-        in_fluxes = in_fluxes.round(digits)
+        out_fluxes = out_fluxes.sort_values(
```

Comment:
I don't see why this is necessary.

It looks to be less efficient than the inplace call.